doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -43,9 +43,13 @@ commands that operate on pname:instance or a child of pname:instance can: be
 obtained. The returned function pointer must: only be called with a
 dispatchable object (the first parameter) that is a child of pname:instance.
 
-If pname:pName is not the name of a core Vulkan command, or is an
-extension command for any extension not supported by any available layer or
-implementation, then fname:vkGetInstanceProcAddr will return `NULL`.
+If pname:pName is not the name of one of these Vulkan commands, and is not
```

Comment:
Right, it's the intervening paragraph that makes "these Vulkan commands" fuzzier.

instance==NULL should only apply to EnumerateInstance* and CreateInstance.
instance== valid VkInstance applies to everything else (all core Vulkan commands as well as enabled instance extension functions.)

How about:
If pname:pName is not a core Vulkan command and is not the name of an enabled instance extension command, then fname:vkGetInstanceProcAddr must: return 'NULL'.